### PR TITLE
Database connection added for jobs

### PR DESCRIPTION
**Pull request by amrathmd**

### CODE REVIEW

**Comment by amrathmd** [comment]
Screenshot from 2023-02-14 12-31-01

File: src/components/DataModels/JobDet.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-type Company={
+{/*type Company={
```
Comment (maharjunm) [inline src/components/DataModels/JobDet.tsx]: remove commented code.

File: src/pages/Home/Home.tsx [hunk 2]
```diff
@@ -16,7 +15,18 @@ const Home = () => {
     setView(currentView);
     setCurrentJob(job);
   };
-  const jobs=JobsData;
+  const [job,setJobs]=React.useState([]);
+  React.useEffect(()=>{
+    const fetchData=async()=>{
+      await axios.get('http://localhost:3000/v1/job')
```
Comment (maharjunm) [inline src/pages/Home/Home.tsx:21]: read ip details from .env file.

File: src/pages/Jobs/Jobs.tsx [hunk 3]
```diff
@@ -1,10 +1,27 @@
-import React from 'react';
+import React, { ButtonHTMLAttributes } from 'react';
 import { ErrorBoundary } from '../../components';
+import axios  from 'axios';
+import jobListings from './data';
 
 const Jobs = () => {
+  async function Test(event:React.FormEvent<HTMLFormElement>){
```
Comment (maharjunm) [inline src/pages/Jobs/Jobs.tsx:7]: remove this code.

File: src/pages/Jobs/data.tsx [hunk 4]
```diff
@@ -0,0 +1,126 @@
+const jobListings = [
```
Comment (maharjunm) [inline src/pages/Jobs/data.tsx:1]: remove this code.

**Pull request closed without merging**